Corona-Warn-App/src/main/java/de/rki/coronawarnapp/environment/EnvironmentSetup.kt

```diff
@@ -36,6 +38,8 @@ class EnvironmentSetup @Inject constructor(
         VERIFICATION_KEYS("PUB_KEYS_SIGNATURE_VERIFICATION"),
         DATA_DONATION("DATA_DONATION_CDN_URL"),
         LOG_UPLOAD("LOG_UPLOAD_SERVER_URL"),
+        VACCINATION_PROOF("VACCINATION_PROOF_SERVER_URL"),
```

Comment:
Please also extend `prod_environments.json` with a placerholder and create a PR on our internal repo to add placeholders/keys to our `test_environments.json`.

Reply:
When using values, use something that satisfies `requireValidUrl` e.g. `https://placeholder`

Reply:
adjusted.
will make a follow up pr for test_env -> internal repo